website/docs/r/spring_cloud_dynatrace_application_performance_monitoring.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_dynatrace_application_performance_monitoring"
+description: |-
+  Manages a Spring Cloud Application Performance Monitoring resource for Dynatrace.
+---
+
+# azurerm_spring_cloud_dynatrace_application_performance_monitoring
+
+-> **NOTE:** This resource is only applicable for Spring Cloud Service enterprise tier
+
+Manages a Spring Cloud Application Performance Monitoring resource for Dynatrace.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_spring_cloud_service" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "E0"
+}
+
+resource "azurerm_spring_cloud_dynatrace_application_performance_monitoring" "test" {
+  name                    = "acctest-apm-%[2]d"
+  spring_cloud_service_id = azurerm_spring_cloud_service.test.id
+  globally_enabled        = true
+  api_url                 = "https://test-api-url.com"
+  api_token               = "test-api-token"
+  environment_id          = "test-environment-id"
+  tenant                  = "test-tenant"
+  tenant_token            = "test-tenant-token"
+  connection_point        = "test-connection-endpoint"
+}
```

Comment:
Can we correct the template here (which seems directly copied from acctest), e.g. the `%[2]d` things, referencing and the naming?